nutanix/resource_nutanix_ndb_database.go

```diff
@@ -215,6 +215,70 @@ func resourceDatabaseInstance() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
+						"ha_availability": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"cluster_name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"patroni_cluster_name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"proxy_read_port": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"proxy_write_port": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"provision_virtual_ip": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  true,
+									},
+									"deploy_haproxy": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  true,
+									},
+									"enable_synchronous_mode": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  true,
+									},
+									"failover_mode": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"node_type": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Default:  "database",
+									},
+									"archive_wal_expire_days": {
+										Type:     schema.TypeInt,
+										Optional: true,
+										Default:  -1,
+									},
+									"backup_policy": {
```

Comment:
this can have choices prefer_secondary, secondary. Lets discuss

Reply:
as discussed , keeping default as primary_only.